docs/containers.conf.5.md

```diff
@@ -10,12 +10,16 @@ and modify the defaults for running containers on the host. containers.conf uses
 a TOML format that can be easily modified and versioned.
 
 Container engines read the /usr/share/containers/containers.conf and
-/etc/containers/containers.conf files if they exists.  When running in rootless
-mode, they also read $HOME/.config/containers/containers.conf files.
+/etc/containers/containers.conf, and /etc/containers/containers.conf.d/* files
+if they exists.  When running in rootless mode, they also read
+$HOME/.config/containers/containers.conf and
+$HOME/.config/containers/containers.conf.d/* files.
```

Comment:
Could we limit it to `*.conf` files as we do for `registries.conf.d`?

Reply:
Can you elaborate on the order of root and rootless?

When running as a rootless user which files/paths in /etc/ do we read?

Reply:
All of them,

Reply:
Man page says
```Container engines read the /usr/share/containers/containers.conf and
/etc/containers/containers.conf, and /etc/containers/containers.conf.d/* files
if they exists.  When running in rootless mode, they also read
$HOME/.config/containers/containers.conf and
$HOME/.config/containers/containers.conf.d/* files.
```
Notice the `also`, how should I make this more obvious?

Reply:
I wasn't aware that /etc/ is loaded when files in $HOME exist.  The current description seems good, thanks!